common/device-model.cpp

```diff
@@ -3376,93 +3395,126 @@ namespace rs2
                 }
                 if (ImGui::IsItemHovered())
                 {
-                    std::string tooltip = rsutils::string::from()
-                        << "On-Chip Calibration"
-                        << (streaming ? " (Disabled while streaming)" : "");
+                    std::string tooltip;
+                    if( is_d555 )
+                        tooltip = "This will improve the depth noise.\n"
+                                  "Point at a scene that normally would have > 50 %% valid depth pixels,\n"
```

Comment:
double %% ?

Reply:
`%%` is a special, single, char to print `%`. It is useful when printing the string using `printf` like functions where % is a special case.
The string was copied from D400, no changes made.